test/plots/barPlotTests.ts

```diff
@@ -773,6 +773,23 @@ describe("Plots", () => {
         assert.include(["visible", "inherit"], label2.style("visibility"), "label 1 is visible");
         svg.remove();
       });
+
+      it("hides labels that are partially cut off in y", () => {
+        yScale.domain([1, 2]);
+        let texts = barPlot.content().selectAll("text");
+
+        assert.strictEqual(texts.size(), 2, "There should be two labels rendered");
```

Comment:
Can we replace the 2 to be aligned with our input?

Maybe count the number of datum we have.

@jtlan However this does mean that if the number of texts is 0, everything would pass.  Should we also assert that `texts.size()` is greater than 0?